wdio.conf.js

```diff
@@ -0,0 +1,182 @@
+// Stop node from complaining about fake memory leaks at higher concurrency
+require("events").defaultMaxListeners = 20;
+require("dotenv").config();
+
+const path = require("path");
+const uuid = require("uuid/v4");
+const localIdentifier = uuid();
+
+const screenResolution = "1920x1080";
+const browserstack = require("browserstack-local");
+
+const defaultCapabilities = {
+  "browserstack.debug": true,
+  "browserstack.local": true,
+  "browserstack.networkLogs": true,
+  "browserstack.localIdentifier": localIdentifier
+};
+
+const desktopCapabilities = {
+  ...defaultCapabilities,
+  os: "windows",
+  os_version: "10",
+  resolution: screenResolution
+};
+const windows7Capabilities = {
+  ...defaultCapabilities,
+  os: "windows",
+  os_version: "7",
+  resolution: screenResolution
+};
+
+const mobileCapabilities = {
+  ...defaultCapabilities,
+  real_mobile: true,
+  "browserstack.appium_version": "1.14.0"
+};
+
+let capabilities = [
+  {
+    ...desktopCapabilities,
+    browserName: 'Google Chrome',
+    browser: "chrome",
+    browser_version: "76",
```

Comment:
my gut tells me that maintaining this _could_ be annoying. is there something more browserslist-like that we could use? like idk "latest" or "latest-1" or something?

Reply:
I think we can just leave off the version to get the latest. Will do that.

Reply:
Looks like Edge and IE require a version, the rest can run against the latest.